### DISCUSSION TITLE
[Multiapp][Turn off subapp]

### DISCUSSION THREAD

**Discussion by coskrrb2002** [idaholab/moose#20901] [Q&A General] [answered]
Dear users,

Is there any way to turn off subapp in the mainapp at the certain time or at the certain value?

Cheers

**Comment by loganharbour** [comment, marked as answer]
You can use the Controls system (https://mooseframework.inl.gov/moose/syntax/Controls/) to change the `enabled` flag on a MultiApp.

**Reply by coskrrb2002** [reply]
Thank you for your advice!
